book/src/concepts/what-is-a-transaction.md

```diff
@@ -29,6 +29,31 @@ Transactions consist of:
 - a gas object - the `Coin` object used to pay for the transaction;
 - gas price and budget - the cost of the transaction;
 
+<sup>* Notice that custom structures defined in Move packages cannot be created and passed to
+MoveVM as transaction inputs. They always need to be instantiated/created inside MoveVM. There are
+only some exceptions to this such as: `ID` and `String`</sup>
+
+## Inputs
+
+Transaction inputs are the arguments for the transaction and are split between 2 types:
+- Pure arguments: These are mostly [primitive types](../move-basics/primitive-types.html) with some
+extra additions. A pure argument can be:
+    - boolean
+    - integer (u8, u16, u32, u64, u128, u256)
+    - address
+    - std::ascii::String
```

Comment:
I am not really sure for this. @damirka , can you confirm?
Also have I left out any type that can be a pure argument?